buildchow/templates/core-jobs.yaml

```diff
@@ -5,9 +5,11 @@
       master
     devstudio-stream:
       master
-    target-platform-version: 4.50.0.Beta1-SNAPSHOT
-    target-platform-version-maximum: 4.50.0.Beta1-SNAPSHOT
-    disabled: true
+    target-platform-version: 4.50.0.CR1-SNAPSHOT
+    target-platform-version-maximum: 4.50.0.CR1-SNAPSHOT
+    blockBuildWhenDownstreamBuilding: false
```

Comment:
Just not following why this needs to be here when the component jobs don't need to refer to it at all.

And I do not follow why it would be hard to templetize with dependencies; this would just be a mention in the list of components what component they rely on. jjb should be possible to handle that pretty well IMO.